src/zkey/pvsecrets.c

```diff
@@ -0,0 +1,55 @@
+/*
+ * zkey - Generate, re-encipher, and validate secure keys
```

Comment:
Hm, this looks like as if it was copied from zkey, but it was not, the content looks entirely different: https://github.com/ibm-s390-linux/s390-tools/blob/master/zkey/pvsecrets.c

I unterstand that you don't want to re-invent the wheel, but copying source from zkey only makes sense when the zkey source are unmodified, otherwise the run out of sync very quickly.

Reply:
Yes right ... the code here is in fact libzpc specific. So I would just change the comment.

Reply:
and maybe also the source location? It currently is inside the zkey subdirectory....

Reply:
Comment changed in both, pvsecrets.c and .h

Reply:
> and maybe also the source location? It currently is inside the zkey subdirectory....
Hm, as pvsecret is a third key type besides CCA and EP11, I placed the new files here. I think this is the natural location for them. Where would you put them alternatively?

Reply:
I would only place it into the zkey subdirectory if it is related to zkey. But since it is not, it does not belong here. I would just place it into `src` directly.

Reply:
OK, done.